### PR TITLE
Add kturtle package

### PR DESCRIPTION
**Pull request by EgorMatirov**

### CODE REVIEW

File: bacs/kturle/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Maintainer: Mikhail Ivchenko <ematirov@gmail.com>
+pkgname='kturtle.interpreter-git'
```
Comment (sarum9in) [inline bacs/kturle/PKGBUILD]: Let's rename it to `bacs.kturtle-git` or `bacs.kturtle.interpreter-git` (something with `bacs` prefix).

File: bacs/kturle/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+pkgver=943.2fcaeef
+pkgrel=1
+pkgdesc="Kturtle customized for BACS"
+arch=('i686' 'x86_64')
+url="https://github.com/EgorMatirov/kturtle"
+license=('GPL' 'LGPL')
```
Comment (sarum9in) [inline bacs/kturle/PKGBUILD:8]: In project it is `GPL` only. Where does `LGPL` come from?

File: bacs/kturle/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+source=("$pkgname::git+https://github.com/EgorMatirov/kturtle.git")
+noextract=()
+md5sums=('SKIP')
+
+pkgver() {
+  cd "$srcdir/$pkgname"
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
```
Comment (sarum9in) [inline bacs/kturle/PKGBUILD:28]: While it does not belong to this PR, please rename binary to `bacs.kturtle` or something with `bacs` prefix. https://github.com/EgorMatirov/kturtle/commit/24d4c2c0f0deb3fac93a2f00b348981b9fab34da is the rename commit but I would strongly prefer this fork to mention `bacs`.

**Review by sarum9in — CHANGES_REQUESTED** [review]

File: bacs/kturle/PKGBUILD (hunk 2 above)
Comment (EgorMatirov) [reply bacs/kturle/PKGBUILD:8]: Copied it from here:
https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/kturtle
Looks strange anyway.

File: bacs/kturle/PKGBUILD (hunk 3 above)
Comment (EgorMatirov) [reply bacs/kturle/PKGBUILD:28]: Done. https://github.com/EgorMatirov/kturtle/commit/301f66c5ecf18263f9a163c5f5798de733c6352a

**Review by sarum9in — APPROVED** [review]
